package.json

```diff
@@ -113,6 +113,7 @@
     "assert": "1.5.0",
     "async-mutex": "0.3.2",
     "asyncstorage-down": "4.2.0",
+    "big-integer": "1.6.51",
```

Comment:
Needed this package to polyfill `BigInt` for `formatByCoinType` in `@ensdomains/address-encoder`